src/main/java/cofh/thermaldynamics/duct/attachments/servo/ServoFluid.java

```diff
@@ -84,8 +84,14 @@ public void tick(int pass) {
 			return;
 		}
 
-		maxInput = ductHandler.fill(getTheTile().drain(maxInput, false), false);
-		FluidStack returned = getTheTile().drain(maxInput, true);
+		IFluidHandler tileHandler = getTheTile();
+
+		if(tileHandler == null) {
```

Comment:
Context: Some tiles might not have a IFluidHandler capability and therefore return null.